build-tests/api-extractor-test-05/etc/api-extractor-test-05.api.json

```diff
@@ -27,142 +27,492 @@
               "canonicalReference": "(deprecatedExample:instance,0)",
               "docComment": "/**\n * @deprecated\n *\n * Use `otherThing()` instead.\n */\n",
               "isStatic": false,
-              "resultTypeSignature": "void",
               "releaseTag": "Public",
               "overloadIndex": 0,
               "parameters": [],
-              "signature": "deprecatedExample(): void;"
+              "excerptTokens": [
+                {
+                  "kind": "Reference",
+                  "text": "deprecatedExample"
+                },
+                {
+                  "kind": "Content",
+                  "text": "(): "
+                },
+                {
+                  "kind": "Content",
+                  "text": "void"
+                },
+                {
+                  "kind": "Content",
+                  "text": ";"
+                }
+              ],
+              "embeddedExcerptsByName": {
+                "ReturnType": {
+                  "startIndex": 2,
+                  "endIndex": 3
+                }
+              }
             },
             {
               "kind": "Method",
               "name": "exampleFunction",
               "canonicalReference": "(exampleFunction:instance,0)",
               "docComment": "/**\n * This is an overloaded function.\n *\n * @param a - the first string\n *\n * @param b - the second string\n */\n",
               "isStatic": false,
-              "resultTypeSignature": "string",
               "releaseTag": "Public",
               "overloadIndex": 0,
               "parameters": [
                 {
                   "kind": "Parameter",
                   "name": "a",
                   "canonicalReference": "a",
-                  "resultTypeSignature": "string",
-                  "signature": "a: string"
+                  "excerptTokens": [
+                    {
+                      "kind": "Reference",
+                      "text": "a"
+                    },
+                    {
+                      "kind": "Content",
+                      "text": ": "
+                    },
+                    {
+                      "kind": "Content",
+                      "text": "string"
+                    }
+                  ],
+                  "embeddedExcerptsByName": {
+                    "ParameterType": {
```

Comment:
>ParameterType [](start = 21, length = 13)

possibly capitalized wrong. if this is a fixed list then this is more like an interface than a dictionary key